Makefile

```diff
@@ -79,6 +89,10 @@ set_env_class_stage:
 	$(eval export ENABLE_AUTO_DEPLOY=true)
 	$(eval export DISABLE_AUTODELETE=1)
 	$(eval export PAAS_CF_TAG_FILTER=stage-*)
+	$(eval export ROOT_DNS_ZONE=cloudpipeline.digital)
+	$(eval export APPS_ROOT_DNS_ZONE=cloudpipelineapps.digital)
+	$(eval export ENV_DNS_ZONE=${AWS_ACCOUNT}.${ROOT_DNS_ZONE})
```

Comment:
See my comment below about naming of variables. I think `ENV` is somewhat ambiguous here. It can be interpreted to mean values that come from environment variables.

I'm not sure we need the extra layer of indirection. We can just set the apps and system domain names here.  Do we even need the `ROOT_DNS_ZONE` variable? I think it's worth avoiding having more variables here than we need.